chapter09/mdw/Ex05.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+$^I= ".out";
+
+my $line = 1;
+
+while (<>) {
+	if ( $line == 2 ) {
+		if ( !(/\#\# Copyright \(C\) 2015 by mdw/) ){
+			s/(.*)/## Copyright (C) 2015 by mdw\n$1/;
+		};
```

Comment:
Yeah, I don't know why I put the ";" there. I don't think it affects how code is run it just a style thing. I don't know if there is a recommendation on if it is helpful/confusing.